### ISSUE TITLE
Testing with constant number of users

### ISSUE THREAD

**Issue by tolgaatam** [shoreditch-ops/artillery#370]
Hi,

I am nowadays testing my API server via Artillery and looking for a way to have constant number of users so that i am able to simulate a more natural behaviour of users. how can i create scenarios so that there will be constant amount of users during the whole test.

I know that this is not an issue but stackoverflow is not much active in terms of Artillery discussions. Any help will be appreciated.

Thanks!

**Comment by hassy** [comment]
Something like this @tolgaatam: #353, i.e. cap the max number of users and create new ones as needed to keep that number constant? Or do you have something different in mind?

**Comment by tolgaatam** [comment]
yes, something like that would work. I am testing a small api but number of users run up to more than hundreds really quickly, hence my test does not provide smallish cases that i would stick to. having a maximum concurrent user cap makes sense for this case
